src/components/DataSourceForm/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Typography } from "antd";
+
+import FormTile from "../FormTile";
+import SearchInput from "../SearchInput";
+
+import { dbTiles } from "./data";
+
+import styles from "./index.module.less";
+
+import type { FC } from "react";
+
+const { Title, Text } = Typography;
+
+const DataSourceForm: FC = () => {
+  const [activeTile, setActiveTile] = useState<string>();
+  const [keyword, setKeyword] = useState<string>("");
+
+  return (
+    <div className={styles.wrapper}>
+      <Title level={3}>New Data Source</Title>
```

Comment:
let's use https://react.i18next.com/latest/usetranslation-hook for all strings, we'll need to connect multiple languages